test/pythonml/test_context.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import re
+import unittest
+from unittest import mock
+
+
+class TestContext(unittest.TestCase):
+    """
+    Unit tests for the methods in the Context class defined in the settings.py module
+    """
+
+    def setUp(self):
+        with open('fixtures/settings.py', 'r') as file:
+            unmodified_settings_file = file.readlines()
+        with open('settings.py', 'w') as file:
```

Comment:
You can collapse both of these with-statements into a single one. For example:

```python
with open('fixtures/settings.py', 'r') as unmodified_settings_file, open('settings.py', w') as rendered_settings_file:
   ...
```

Reply:
Decided to leave this as having both, if that's ok